doc/qbk/2.0.quicklook.qbk

```diff
@@ -59,36 +66,36 @@ Say you have the following URL that you want to parse:
 [c++]
 [snippet_parsing_1]
 
-You can parse it by calling this function:
+[note
+    The library namespace includes the aliases __string_view__, __error_code__, and __result__.
+]
 
-[snippet_parsing_2]
+Note that __string_view__ is an alias to `boost::core::string_view`, a
+`string_view` implementation that is implicitly convertible to
+`std::string_view`.
 
-[note
-    Sample code and identifiers used throughout are written as if
-    the following declarations are in effect:
+You can parse the string by calling this function:
 
-    [snippet_headers_3]
-]
+[snippet_parsing_2]
 
 The function __parse_uri__ returns an object of type `__result__<__url_view__>`
 which is a container resembling a variant that holds either an error or an object.
+A number of [link section.url_view functions] are available to parse different
+types of URL.
 
 We can immediately call `result::value` to obtain a __url_view__.
 
 [snippet_parsing_3]
 
-`result::value` throws an exception on a parsing error.
-
-[snippet_parsing_4]
-
-Alternatively, the functions `result::has_value` and
-`result::has_error` could also be used to check if
-the string has been parsed without errors.
-
-[snippet_parsing_5]
+When there are no errors, [@boost:/libs/system/doc/html/system.html#ref_checked_value_access_2 `result::value`]
+returns an instance of __url_view__, which holds the parsed result.
+[@boost:/libs/system/doc/html/system.html#ref_checked_value_access_2 `result::value`]
+throws an exception on a parsing error.
 
-When there are no errors, it `result::value` returns an instance of __url_view__,
-which holds the parsed result.
+Alternatively, the functions
+[@boost:/libs/system/doc/html/system.html#ref_queries `result::has_value`] and
+[@boost:/libs/system/doc/html/system.html#ref_queries `result::has_error`] could
+also be used to check if the string has been parsed without errors.
 
 [note
     It is worth noting that __parse_uri__ allocates no memory and throws no exceptions.
```

Comment:
This is not true. According to synopsis, this function throws `length_error` when  the input string is too long. 
I think that the essential guarantee is that:
* The function does not allocate memory
* The situation where the input does not match the URL grammar are reporter through `result<>` rather than exceptions
* Any other errors (such as excessive length) are reported via exceptions.

Reply:
There is another issue with this note. Because `url_view` is non-owing, it cannot guarantee that it always represents a valid URI, because the contents of the original string may be changed externally:
https://godbolt.org/z/x4nzGPb9G